### DISCUSSION TITLE
bench result is not expected

### DISCUSSION THREAD

**Discussion by Kyofin** [juicedata/juicefs#2533] [Q&A]
I use mysql and redis as juicefs metadata ,use minio as stroage , i found the some items are not good of the  bench result .
How to improve the performance ?

metadata: redis 6.0.9  
storage: minio RELEASE.2022-08-02T23-59-16Z
<img width="1440" alt="image" src="https://user-images.githubusercontent.com/18548053/184787408-4ef2644c-d5fa-46c8-9ef2-abd793e12815.png">

---
metadata: mysql 5.7
storage: minio RELEASE.2022-08-02T23-59-16Z
<img width="905" alt="image" src="https://user-images.githubusercontent.com/18548053/184787459-1f3d0c51-1068-4e0c-859d-458d83177dc2.png">

**Comment by xiaogaozi** [comment]
The latency of "Put object" and "Get object" is really too high, usually the latency of accessing object storage on public cloud is around tens of milliseconds. Are JuiceFS mount point and MinIO deployed on the same network?

**Comment by Antinomy20001** [comment]
hey what hardware environment do you use
